curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/653883da4a1fabaeb5f1f5e7.md

```diff
@@ -0,0 +1,248 @@
+---
+id: 653883da4a1fabaeb5f1f5e7
+title: Step 5
+challengeType: 0
+dashedName: step-5
+---
+
+# --description--
+
+The `.getMonth()` method returns a number between 0 and 11 which represents the month for the date provided, where 0 is January and 11 is December.  Because the number this method returns is zero-based, you need to add 1 to it to get the expected month number.
```

Comment:
```suggestion
The `.getMonth()` method returns a number between 0 and 11. This represents the month for the date provided, where 0 is January and 11 is December.  Because the number this method returns is zero-based, you need to add 1 to it to get the expected month number.
```